Sources/ClientRuntime/Networking/Http/URLSession/FoundationStreamBridge.swift

```diff
@@ -42,10 +46,14 @@ class FoundationStreamBridge: NSObject, StreamDelegate {
     let readableStream: ReadableStream
 
     /// A Foundation stream that will carry the bytes read from the readableStream as they become available.
-    let inputStream: InputStream
+    ///
+    /// May be replaced if needed by calling the `replaceStreams(_:)` method.
+    var inputStream: InputStream
 
-    /// A Foundation `OutputStream` that will read from the `ReadableStream`
-    private let outputStream: OutputStream
+    /// A Foundation `OutputStream` that will read from the `ReadableStream`.
+    ///
+    /// Will be replaced when `replaceStreams(_:)` is called to replace the input stream.
+    private var outputStream: OutputStream
```

Comment:
Input & output Foundation streams are now `var`s because they may need to be replaced during a HTTP request.